### PR TITLE
feat(BUX-000): removes PruneTip endpoint and references

### PR DESCRIPTION
**Pull request by jakubmkowalski**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/pulse/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

Removes PruneTip (which has the status "TO BE IMPLEMENTED") before 1.0 release.
Created task with correct implementation of prune tip - BUX-513.

### CODE REVIEW

File: transports/http/endpoints/api/tips/tips_endpoints_test.go [hunk 1]
```diff
@@ -127,52 +126,52 @@ func TestGetTipLongest(t *testing.T) {
...
-		// then
-		assert.Equal(t, res.Code, expected_result.code)
-
-		var tip string
-		json.NewDecoder(res.Body).Decode(&tip)
-
-		assert.Equal(t, tip, expected_result.body)
-	})
-}
+// func TestPruneTip(t *testing.T) {
```
Comment (wregulski) [inline transports/http/endpoints/api/tips/tips_endpoints_test.go:129]: I would remove test as it will be not useful after refactor

Comment (jakubmkowalski) [reply transports/http/endpoints/api/tips/tips_endpoints_test.go]: removed

File: transports/http/endpoints/api/tips/endpoints.go [hunk 2]
```diff
@@ -63,24 +62,15 @@ func (h *handler) getTipLongestChain(c *gin.Context) {
...
-//	@Success 200 {object} string
-//	@Router /chain/tip/prune/{hash} [get]
-//	@Param hash path string true "Requested Header Hash"
-//	@Security Bearer
-func (h *handler) pruneTip(c *gin.Context) {
-	param := c.Param("hash")
-	fmt.Println(param)
-	tip, err := h.service.GetPruneTip()
+// TO BE IMPLEMENTED
+// func (h *handler) pruneTip(c *gin.Context) {
```
Comment (kuba-4chain) [inline transports/http/endpoints/api/tips/endpoints.go:66]: Maybe we could remove this completely and just add a task in JIRA to implement such endpoint? This code will not be very useful anyways :)

File: transports/http/endpoints/api/tips/endpoints.go [hunk 3]
```diff
@@ -26,7 +25,7 @@ func (h *handler) RegisterApiEndpoints(router *gin.RouterGroup, cfg *config.HTTP
 	{
 		tip.GET("/tip", h.getTips)
 		tip.GET("/tip/longest", h.getTipLongestChain)
-		tip.GET("/tip/prune/:hash", h.pruneTip)
+		// tip.GET("/tip/prune/:hash", h.pruneTip)
```
Comment (kuba-4chain) [inline transports/http/endpoints/api/tips/endpoints.go:28]: The same, maybe we could just get rid of it and add JIRA task?

File: transports/http/endpoints/api/tips/endpoints.go (hunk 2 above)
Comment (jakubmkowalski) [reply transports/http/endpoints/api/tips/endpoints.go]: removed

File: transports/http/endpoints/api/tips/endpoints.go (hunk 3 above)
Comment (jakubmkowalski) [reply transports/http/endpoints/api/tips/endpoints.go]: removed

**Review by kuba-4chain — APPROVED** [review]

**Review by Nazarii-4chain — APPROVED** [review]

**Pull request merged**